bin/package.js

```diff
@@ -109,11 +109,33 @@ var linux = {
   // Note: Application icon for Linux is specified via the BrowserWindow `icon` option.
 }
 
-electronPackager(Object.assign({}, all, darwin), done)
-electronPackager(Object.assign({}, all, win32), done)
-electronPackager(Object.assign({}, all, linux), done)
+var platform = process.argv[2]
+
+if (platform === '--darwin') {
+  buildDarwin()
+} else if (platform === '--win32') {
+  buildWin32()
+} else if (platform === '--linux') {
+  buildLinux()
+} else {
+  // Build all
+  buildDarwin(() => buildWin32(() => buildLinux()))
+}
+
+function buildDarwin (cb) {
+  electronPackager(Object.assign({}, all, darwin), done.bind(null, cb))
+}
+
+function buildWin32 (cb) {
+  electronPackager(Object.assign({}, all, win32), done.bind(null, cb))
+}
+
+function buildLinux (cb) {
+  electronPackager(Object.assign({}, all, linux), done.bind(null, cb))
+}
```

Comment:
d'oh, I was just working on making the packager accept specific target platforms.